src/wp-includes/rest-api.php

```diff
@@ -491,10 +491,10 @@ function rest_handle_deprecated_function( $function, $replacement, $version ) {
 		return;
 	}
 	if ( ! empty( $replacement ) ) {
-		/* translators: 1: function name, 2: WordPress version number, 3: new function name */
+		/* translators: 1: function name, 2: ClassicPress version number, 3: new function name */
```

Comment:
Let's say "ClassicPress or WordPress version number" for now.  We'll prefix these version numbers with `WP-` if they are WordPress versions.  See #27.